### PR TITLE
JIRA123: Edit caption to describe image

### LINKED ISSUE #2: Activity 1: Your First Caption
Using the API, I have just created a file with a random octocat for you to caption. In this activity, we will learn how to make changes to files in our GitHub repository.

- [ ] @ello21
- [ ] @ericalloyd

### PR DESCRIPTION
**Pull request by ericalloyd**
This PR fixes #2 
Keywords: closes, resolves

Additional images and context.

This PR includes the following tasks:

- [x] Task 1
- [x] Task 2
- [ ] Task 3

### CODE REVIEW

**Review by stoe — APPROVED** [review]
LGTM :shipit:

**Comment by ericalloyd** [comment]
Hey, I'm glad you're working on this effort 👍🏿

File: _slides/03-ericalloyd.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Welcome and Arigato!
```
Comment (ello21) [inline _slides/03-ericalloyd.md]: LGTM, I like that this is inclusive!

Comment (ello21) [inline _slides/03-ericalloyd.md]:
```suggestion
Welcome and Arigato 👋🏾 
```

**Review by ello21 — APPROVED** [review]
This looks good, but I made a suggestion.

File: _slides/03-ericalloyd.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Welcome and Arigato!
+Welcome and Arigato 👋🏾
```
Comment (stoe) [inline _slides/03-ericalloyd.md:6]: [スゴイ](https://www.deepl.com/translator#ja/en/%E3%82%B9%E3%82%B4%E3%82%A4) 👍

**Pull request merged**